.gitmodules

```diff
@@ -33,7 +33,7 @@
 	url = https://github.com/aspnet/websdk
 [submodule "src/msbuild"]
 	path = src/msbuild
-	url = https://github.com/Microsoft/msbuild
+	url = https://github.com/cdmihai/msbuild
```

Comment:
Master does not yet support the sourcebuild parameters, I was thinking to send that PR out when sourcebuild approaches master, but I guess I can just open the PR to our master now.

On another note, master does not work CLIs older than 2.1.3xx, should you set the gitmodule to point to https://github.com/Microsoft/msbuild/tree/sourceBuild_2.1.101?